src/stores/game.js

```diff
@@ -14,6 +14,31 @@ function queenCount(player) {
   return player.faceCards.reduce((queenCount, card) => queenCount + (card.rank === 12 ? 1 : 0), 0);
 }
 
+class GameCard {
+  constructor(card) {
+    const str_rank =
+      {
+        1: 'A',
+        11: 'J',
+        12: 'Q',
+        13: 'K',
+      }[card.rank] ?? card.rank;
+    // Stringify Suit
+    const str_suit = ['♣️', '♦️', '♥️', '♠️'][card.suit];
+    this.createdAt = card.createdAt;
+    this.updatedAt = card.updatedAt;
+    this.id = card.id;
+    this.suit = card.suit;
+    this.rank = card.rank;
+    this.name = str_rank + str_suit;
+    this.attachments = card.attachments;
+  }
+}
+const createGameCard = (card) => {
+  if (!card) return null;
+  return new GameCard(card);
+};
```

Comment:
I'm unsure if this is a good way to do this, without this.. the ui shows some stuff that it shouldn't, specifically when looking for a `target` which = `store.oneOffTarget`, if we just run `if(!card) return` inside the constructor while the is `store.oneOffTarget = null` it becomes `{}` instead of `null` which causes ui elements to show on the `v-if="target"`

Reply:
I think this is reasonable to keep the `null` cards e.g. `topCard` when it's missing to be `null` like they currently are